bazel/thirdparty/Cargo.toml

```diff
@@ -6,7 +6,7 @@ edition = "2021"
 publish = false
 
 [lib]
-path = "/dev/null"
+path = "fake.rs"
```

Comment:
seems to resolve the error below. 

but, i had to expunge the cache for it to help. so it kinda seems like Cargo.toml is not hooked up as a dependency that'll invalidate whatever it is being used to generate, either on our side or in create.from_cargo.

```
ERROR: /home/nwatkins/.cache/bazel/_bazel_nwatkins/31f1aa7012681f681e8d49a1818a8564/external/rules_rust+/crate_universe/extensions.bzl:363:17: Traceback (most recent call last):
        File "/home/nwatkins/.cache/bazel/_bazel_nwatkins/31f1aa7012681f681e8d49a1818a8564/external/rules_rust+/crate_universe/extensions.bzl", line 459, column 37, in _crate_impl
                _generate_hub_and_spokes(
        File "/home/nwatkins/.cache/bazel/_bazel_nwatkins/31f1aa7012681f681e8d49a1818a8564/external/rules_rust+/crate_universe/extensions.bzl", line 144, column 16, in _generate_hub_and_spokes
                cargo_bazel([
        File "/home/nwatkins/.cache/bazel/_bazel_nwatkins/31f1aa7012681f681e8d49a1818a8564/external/rules_rust+/crate_universe/extensions.bzl", line 363, column 17, in run
                fail("%s returned with exit code %d:\n%s" % (pretty_args, result.return_code, result.stderr))
Error in fail: /home/nwatkins/.cache/bazel/_bazel_nwatkins/31f1aa7012681f681e8d49a1818a8564/modextwd/rules_rust++crate/cargo-bazel generate --cargo-lockfile /home/nwatkins/src/redpanda/bazel/thirdparty/Cargo.lock --config /home/nwatkins/.cache/bazel/_bazel_nwatkins/31f1aa7012681f681e8d49a1818a8564/modextwd/rules_rust++crate/crates/config.json --splicing-manifest /home/nwatkins/.cache/bazel/_bazel_nwatkins/31f1aa7012681f681e8d49a1818a8564/modextwd/rules_rust++crate/crates/splicing_manifest.json --repository-dir /home/nwatkins/.cache/bazel/_bazel_nwatkins/31f1aa7012681f681e8d49a1818a8564/modextwd/rules_rust++crate/crates --metadata /home/nwatkins/.cache/bazel/_bazel_nwatkins/31f1aa7012681f681e8d49a1818a8564/modextwd/rules_rust++crate/crates/splicing-output/metadata.json --repin --lockfile /home/nwatkins/.cache/bazel/_bazel_nwatkins/31f1aa7012681f681e8d49a1818a8564/modextwd/rules_rust++crate/crates/lockfile.json --nonhermetic-root-bazel-workspace-dir /home/nwatkins/src/redpanda --paths-to-track /home/nwatkins/.cache/bazel/_bazel_nwatkins/31f1aa7012681f681e8d49a1818a8564/modextwd/rules_rust++crate/paths-to-track --warnings-output-path /home/nwatkins/.cache/bazel/_bazel_nwatkins/31f1aa7012681f681e8d49a1818a8564/modextwd/rules_rust++crate/warnings-output-file --cargo /home/nwatkins/.cache/bazel/_bazel_nwatkins/31f1aa7012681f681e8d49a1818a8564/external/rules_rust++rust_host_tools+rust_host_tools/bin/cargo --rustc /home/nwatkins/.cache/bazel/_bazel_nwatkins/31f1aa7012681f681e8d49a1818a8564/external/rules_rust++rust_host_tools+rust_host_tools/bin/rustc returned with exit code 1:
Error: Package "thirdparty" target "thirdparty" had an absolute source path "/dev/null", which is not supported
```

Reply:
Strange